ModUtils/EventUtils.cs

```diff
@@ -76,7 +121,7 @@ public static string EventName(string objectName, EventType eventType, uint subt
         /// <param name="eventCode"></param>
         /// <param name="eventType"></param>
         /// <param name="subtype"></param>
-        public static void AddNewEvent(string objectName, string eventCode, EventType eventType, uint subtype)
+        public static void AddNewEvent(string objectName, string eventCode, EventType eventType, uint subtype, bool asAsm = false)
```

Comment:
same here this param should be added to the docstring

Reply:
done